### ISSUE TITLE
Pages have a transparent background instead of white

### ISSUE THREAD

**Issue by CritLoren** [zen-browser/desktop#3087]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

A lot of websites rely on the browser having a white background for the iframe of the actual tab, Zen has a transparent background however, which means that the dark text on pages that don't have a background colour set is overlapping the default dark background of the browser.

You can notice it on pages such as https://commission.europa.eu/strategy-and-policy/priorities-2019-2024/stronger-europe-world/eu-solidarity-ukraine_en
![image](https://github.com/user-attachments/assets/bd99d574-e373-4d6d-9d68-a60f004eccc9)

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.19

### What platform are you seeing the problem on?

Linux

**Comment by CritLoren** [comment]
Fixed by https://github.com/zen-browser/desktop/issues/3047#issuecomment-2480552539

**Comment by saforem2** [comment]
wow, just as I was getting frustrated enough to look into why this was happening I find this fix. love to see it 🚀

**Comment by Schuasda** [comment]
i am having this issue again on the yazi docs:
https://yazi-rs.github.io/docs/configuration/overview

**Comment by mr-cheffy** [comment]
@Schuasda https://github.com/zen-browser/desktop/issues/3087#issuecomment-2483052000

**Comment by Schuasda** [comment]
so, i opened my browser today and encountered the issue.
I found this ticket and some other fixes, which didn't work
I made sure i had the latest update and i didn't have some mod or extension, that might cause this ( i have not changed my config in the past days/weeks, but still )
Now i double check the transparancy setting from the comment, it is set to false, i restart the browser ( have done so multiple times already )
The issue is gone

tl;dr: either i am going insane or it is some really stupid layer 8 issue. Works now.
